src/mesh.rs

```diff
@@ -101,49 +101,49 @@ pub struct Mesh {
     pub normals: Vec<Normal>,
     pub indices: Vec<usize>,
     pub bounds: BoundingBox,
-    stl_had_normals: bool,
+    model_had_normals: bool,
 }
 
 impl Mesh {
     // Load mesh data from file (if provided) or stdin
-    pub fn load(stl_filename: &str, recalc_normals: bool) -> Result<Mesh, Box<dyn Error>> {
+    pub fn load(model_filename: &str, recalc_normals: bool) -> Result<Mesh, Box<dyn Error>> {
         // TODO: Add support for URIs instead of plain file names
         // https://developer.gnome.org/integration-guide/stable/thumbnailer.html.en
-        match stl_filename {
+        match model_filename {
             "-" => {
                 // create_stl_reader requires Seek, so we must read the entire stream into memory before proceeding.
                 // So I guess this can just consume all RAM if it gets bad input. Hmmm....
-                let mut input_buffer = Vec::new();
-                io::stdin().read_to_end(&mut input_buffer)?;
-                Mesh::from_stl(Cursor::new(input_buffer), recalc_normals)
+                let mut model_buffer = Vec::new();
+                io::stdin().read_to_end(&mut model_buffer)?;
+                Mesh::from_stl(Cursor::new(model_buffer), recalc_normals)
```

Comment:
Here `input_buffer` was fine

Reply:
Agreed. `input_buffer` is better here but the rest of this looks good. Make that change and I will merge.

Reply:
done. thanks.